.travis.yml

```diff
@@ -1,6 +1,6 @@
 language: node_js
 node_js:
-  - "0.10"
+  - "0.12"
```

Comment:
For some reason I got build issues on travis with 0.10 and couldn't be bothered to debug.